### DISCUSSION TITLE
No Metamask or Coinbase wallets in @walletconnect/modal

### DISCUSSION THREAD

**Discussion by js-sapphire** [WalletConnect/walletconnect-monorepo#3739] [Developer Support] [answered]
Hey Team,

How can I include Metamask and Coinbase as options in @walletconnect/modal ? In my integration, I don't find these two wallets when I open the modal? I tried desktopWallets option but what should I provide in links parameter to get these 2 providers?

<img width="363" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/41758940/01d9b829-d855-43f6-a467-a077e836b88d">
<img width="357" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/41758940/cc11df7b-f9f2-47f7-aaf0-e956870a54e3">

**Comment by glitch-txs** [comment, marked as answer]
WalletConnectModal doesn't support browser extension wallets, for this you would like to use Web3Modal SDK isntead.

https://docs.walletconnect.com/web3modal/about

**Comment by araf-hossain** [comment]
I am able to add Metamask to the WalletConnectModal by adding ID's in the "explorerRecommendedWalletIds". But when I am clicking it's not working as expected. It should give a popup from the extension, but currently it's just giving me a QR code. 

I have seen this discussion, just asking do we have any updates? 
https://github.com/orgs/WalletConnect/discussions/3661 

@glitch-txs, do you have any suggestions?
<img width="405" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/19535365/4611c781-b26c-4309-9e9b-6a617ee9135f">

**Reply by glitch-txs** [reply]
this was answered above :)

**Comment by rahulnrt** [comment]
Dear team, I am using the wallet connect model in my React native mobile application but where i seen all wallets except Coinbase can you please clarify the issue because i want to use Coinbase wallet intregation in my mobile app

**Reply by rahulnrt** [reply]
kindly reply

**Reply by glitch-txs** [reply]
Please ask in RN repository
